src/tracking/config.py

```diff
@@ -0,0 +1,12 @@
+upstreams = {
```

Comment:
We should probably have this listed in DB somewhere instead of directly mentioning it here. These can be default but we need to allow user to specify their own third party rpm repositories if they want to.